### ISSUE TITLE
iframes should have accessible names

### ISSUE THREAD

**Issue by Malvoz** [mdo/github-buttons#137]
Google's [Lighthouse](https://github.com/GoogleChrome/lighthouse) is flagging for accessibility issues when [iframe is missing the `title` attribute](https://web.dev/frame-title/) and consequently lowers the website's score. Consider adding appropriate titles such as `title="Star on Github"` to iframes in the documentation. ✌️
